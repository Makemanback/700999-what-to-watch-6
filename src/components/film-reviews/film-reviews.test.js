import React from 'react';
import {render} from '@testing-library/react';
import FilmReviews from './film-reviews';

const reviews = [
  {
    user: {
      id: 15,
      name: 'Kendall'
    },
    rating: 6,
    comment: 'I personally found this movie to be boring. Definitely one of the most boring movies I\'ve ever seen.',
    date: '2021-02-15T08:04:28.658Z',
    reviewId: 1
  },
  {
    user: {
      id: 1,
      name: 'makeman'
    },
    rating: 6,
    comment: 'ewwfd',
    date: '2021-04-02T07:36:03.591Z',
    reviewId: 2
  }
];

test(`Should FilmReviews render correctly`, () => {
  const {container} = render(
        <FilmReviews reviews={reviews} />
  );

  expect(container).toMatchSnapshot();
});
