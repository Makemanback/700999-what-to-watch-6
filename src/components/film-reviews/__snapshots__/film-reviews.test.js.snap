// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FilmReviews render correctly 1`] = `
<div>
  <div
    class="movie-card__reviews movie-card__row"
  >
    <div
      class="movie-card__reviews-col"
    >
      <div
        class="review"
      >
        <blockquote
          class="review__quote"
        >
          <p
            class="review__text"
          >
            I personally found this movie to be boring. Definitely one of the most boring movies I've ever seen.
          </p>
          <footer
            class="review__details"
          >
            <cite
              class="review__user"
            >
              Kendall
            </cite>
            <time
              class="review__date"
              datetime="2021-02-15T11:04:28+03:00"
            >
              February 15th 2021
            </time>
          </footer>
        </blockquote>
        <div
          class="review__rating"
        >
          6
        </div>
      </div>
    </div>
    <div
      class="movie-card__reviews-col"
    >
      <div
        class="review"
      >
        <blockquote
          class="review__quote"
        >
          <p
            class="review__text"
          >
            ewwfd
          </p>
          <footer
            class="review__details"
          >
            <cite
              class="review__user"
            >
              makeman
            </cite>
            <time
              class="review__date"
              datetime="2021-04-02T10:36:03+03:00"
            >
              April 2nd 2021
            </time>
          </footer>
        </blockquote>
        <div
          class="review__rating"
        >
          6
        </div>
      </div>
    </div>
  </div>
</div>
`;
